main/webapp/modules/core/scripts/reconciliation/recon-dialog.js

```diff
@@ -211,4 +214,20 @@ ReconDialog.prototype._onAddStandardService = function() {
   elmts.input.trigger('focus').trigger('select');
 };
 
-
+/**
+ * Refetches the manifest for a reconciliation service and re-registers it.
+ * 
+ * @param {Object} service - The reconciliation service object
+ * @param {Function} [f] - Optional callback function to execute after re-registering
+ */
+ReconDialog.prototype._refetchServiceManifest = function(service, f) {
+  var dismissBusy = DialogSystem.showBusy($.i18n('core-recon/contact-service')+"...");
+  $.ajax({ url: service.url })
```

Comment:
We currently let users add reconciliation services via two means:
* normal requests, like the one you are doing here, which succeed if the service has set the appropriate CORS headers
* via the legacy JSONP method (by adding a `callback` parameter to the GET URL)

I think this refetching will fail if the service was added via JSONP, so perhaps this needs refactoring so this request is done by the ReconciliationManager already (which already handles those two cases).

Eventually we should drop JSONP support, but there are still a few publicly available services which are only accessible via JSONP: https://reconciliation-api.github.io/testbench/